enrollment-server-onboarding/src/main/resources/api/api-iproov.json

```diff
@@ -1032,21 +1027,6 @@
             "liveness"
           ]
         },
-        "success_url": {
-          "description": "The URL to redirect to on success. Note: This field has been deprecated and will be omitted in the next release.",
-          "type": "string",
-          "deprecated": true
-        },
-        "failure_url": {
-          "description": "The URL to redirect to on failure. Note: This field has been deprecated and will be omitted in the next release.",
-          "type": "string",
-          "deprecated": true
-        },
-        "abort_url": {
-          "description": "The URL to redirect to on user abort. Note: This field has been deprecated and will be omitted in the next release.",
-          "type": "string",
-          "deprecated": true
-        },
         "user_id": {
```

Comment:
Manually edited OpenAPI, when download https://eu.rp.secure.iproov.me/swagger.json you get this. But confirmed by iProov, that is not data validation but only recommendation.

```json
"type": "string",
"format": "uuid"
```